generator/source/faq/settings.rst

```diff
@@ -41,9 +41,9 @@ sawtooth.config.authorization_type
     Example setting--never used.  To set authorization type, use command line option ``sawtooth-validator --network-auth {trust|challenge}``
 
 sawtooth.consensus.algorithm.name
-    Pluggable consensus algorithm name. These include ``PoET``, ``Devmode``, ``sawtooth-raft-engine``, and ``sawtooth-pbft``.  The default is ``devmode`` for Sawtooth 1.1 or earlier, with no default for the (unreleased) nightly build.
+    Pluggable consensus algorithm name. These include ``PoET``, ``Devmode``, ``raft``, and ``pbft``.  The default is ``devmode`` for Sawtooth 1.1 or earlier, with no default for the (unreleased) nightly build.
 sawtooth.consensus.algorithm.version
-    Consensus algorithm version. Currently 0.1 for PoET, Devmode, and sawtooth-pbft, and 0.1.0 for sawtooth-raft-engine.
+    Consensus algorithm version. Currently 0.1 for PoET, Devmode, pbft, and raft.
```

Comment:
This is a nitpick, but please change the capitalization to PBFT and Raft, because these are the "generic" names for the consensus algorithms.  (If they were the engine names, they'd need to be in double backticks, as above.) Unfortunately, the engine names are not the same as the generic names.